tests/ui/perfherder/alerts-view/select_alert_framework_test.jsx

```diff
@@ -0,0 +1,85 @@
+import { render } from '@testing-library/react';
+import { noop } from 'lodash';
+import React from 'react';
+
+import AlertsViewControls from '../../../../ui/perfherder/alerts/AlertsViewControls';
+import FilterControls from '../../../../ui/shared/FilterControls';
+
+const testUser = ({ isLoggedIn, isStaff }) => ({
+  username: 'mozilla-ldap/test_user@mozilla.com',
+  isLoggedIn,
+  isStaff,
+  email: 'test_user@mozilla.com',
+});
+
+const testFrameworksDropdown = () => {
+  const mockFrameworkNames = ['c', 'b'];
+  const topFramework = 'top framework';
+
+  const alertDropdowns = [
+    {
+      options: mockFrameworkNames,
+      selectedItem: 'select framework',
+      updateData: () => noop(),
+      namespace: 'framework',
+      pinned: [topFramework],
+      otherPinned: ['select framework', 'bottom framework'],
+    },
+  ];
+
+  return render(
+    <FilterControls
+      filteredTextValue={false}
+      dropdownOptions={alertDropdowns}
+      filterOptions={[]}
+      updateFilter={() => noop()}
+      updateFilterText={() => noop()}
+      updateOnEnter
+      dropdownCol
+    />,
+  );
+};
+
+test('should pin the right number of items to top and bottom frameworks w.r.t config', async () => {
+  testFrameworksDropdown();
+
+  const topPinned = document.querySelectorAll('.top-pinned');
+  const bottomPinned = document.querySelectorAll('.bottom-pinned');
```

Comment:
Could we specify with a comment or in the test name what `topPinned` and `bottomPinned`  represent?
It will make clear why we separate these frameworks in the first place.